src/wizards/deployToOpenShift/deployToOpenShift.ts

```diff
@@ -0,0 +1,61 @@
+import { commands, Uri, window } from "vscode";
+import { ProjectLabelInfo } from "../../definitions/ProjectLabelInfo";
+import { getOpenShiftConnector, installOpenShiftConnectorWithProgress, isOpenShiftConnectorInstalled, OPENSHIFT_CONNECTOR } from "../../utils/openShiftConnectorUtils";
+import { getQuarkusProject } from "../getQuarkusProject";
+
+/**
+ * Attempts to select a Quarkus project, then deploy it to OpenShift using OpenShift Connector
+ */
+export async function deployToOpenShift() {
+  try {
+    await installOpenShiftConnectorIfNeeded();
+    const quarkusProject: ProjectLabelInfo = await getQuarkusProject();
+    const result: string = await commands.executeCommand("openshift.component.createFromRootWorkspaceFolder", //
+      Uri.parse(quarkusProject.uri) as Uri, //
+      [] as Uri[], // unused in OpenShift Connector
+      undefined, // OpenShift context
+      "java-quarkus");
+    if (!result) {
+      throw new Error();
+    }
+    return window.showInformationMessage(result);
+  } catch (error) {
+    return window.showErrorMessage(`Failed to deploy to OpenShift`);
```

Comment:
I need to remove the error/success reporting here, since OpenShift connector will handle this